chat-server/package.json

```diff
@@ -8,10 +8,13 @@
   "type": "module",
   "main": "server.js",
   "scripts": {
-    "start": "node server.js"
+    "start": "nodemon server.js"
```

Comment:
Nice touch using `nodemon`!  This can save you a lot of time and effort during development. Well done!